src/cow-react/modules/limitOrders/pure/LimitOrdersDetails/index.tsx

```diff
@@ -34,32 +33,52 @@ export function LimitOrdersDetails(props: LimitOrdersDetailsProps) {
       </styledEl.DetailsRow>
       <styledEl.DetailsRow>
         <div>
-          <span>Expiry</span> <InfoIcon content={'Expiry info TODO'} />
+          <span>Expiry</span>
+          <InfoIcon
+            content={
+              "If your order has not been filled by this date & time, it will expire. Don't worry - expirations and order placement are free on CoW Swap!"
```

Comment:
I saw that in Explorer, for quote sign we used `&quot;` instead of just `'` so I was wondering if those 2 things are the same and can we just write it as you did here?

Reply:
The editor/linter replaces `'` automatically if it's in the HTML part of the code.
If it's a string it doesn't.

Don't know the details but I assume it's fine when passing a string